From 706af674479c8dad868dee646b809dead6ac9445 Mon Sep 17 00:00:00 2001
From: Eric Koegel <eric.koegel@gmail.com>
Date: Wed, 25 Feb 2015 13:50:39 +0300
Subject: [PATCH 02/12] daemon: remove dead code

In commit 2c51bd83b65076cf98bd1baf48d9add31435d118,
DBusGProxy *bus_proxy was converted to GDBusProxy but the daemon
hasn't needed or used the bus_proxy since the conversion to GDBUS.
This patch simply removes it.

https://bugs.freedesktop.org/show_bug.cgi?id=89318
---
 src/daemon.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/daemon.c b/src/daemon.c
index 01afd07..e62e124 100644
--- a/src/daemon.c
+++ b/src/daemon.c
@@ -57,7 +57,6 @@ enum {
 
 struct DaemonPrivate {
         GDBusConnection *bus_connection;
-        GDBusProxy *bus_proxy;
 
         GHashTable *users;
 
@@ -550,9 +549,6 @@ daemon_finalize (GObject *object)
 
         daemon = DAEMON (object);
 
-        if (daemon->priv->bus_proxy != NULL)
-                g_object_unref (daemon->priv->bus_proxy);
-
         if (daemon->priv->bus_connection != NULL)
                 g_object_unref (daemon->priv->bus_connection);
 
-- 
1.9.5

